lib/alerts/reducers/active.js

```diff
@@ -25,6 +25,14 @@ const active = (state: ActiveState = defaultState, action: Action): ActiveState
     case 'SET_ACTIVE_ALERT_PUBLISHED':
       return update(state, {active: {published: {$set: action.payload}}})
     case 'ADD_ACTIVE_ALERT_AFFECTED_ENTITY':
+      const {affectedEntities} = state.active
+      // Determine and assign non-conflicting id for new entity.
+      const id = affectedEntities && affectedEntities.length
+        ? 1 + affectedEntities
+          .map(e => e.id)
+          .reduce((initial, current) => initial > current ? initial : current)
```

Comment:
Is there a way we can just use a UUID here? I think it'd be a good idea to document what this id is a little better somewhere.

Reply:
Yea, that's probably a better approach, isn't it? I'd need to check that a create alert request didn't fail if it has a string value, but generally I like the idea.